src/nerve_tools/updown_service.py

```diff
@@ -272,8 +362,17 @@ def main() -> None:
     if not args.wait_only:
         reconfigure_hacheck(args.service, args.state, args.port)
 
-    result = wait_for_haproxy_state(
-        args.service, args.state, timeout_s, args.wait_time)
+    if args.envoy_eds_dir:
+        result = wait_for_envoy_state(
+            service=args.service,
+            expected_state=args.state,
+            timeout=timeout_s,
+            wait_time=args.wait_time,
+            envoy_eds_dir=args.envoy_eds_dir,
+        )
+    else:
+        result = wait_for_haproxy_state(
+            args.service, args.state, timeout_s, args.wait_time)
```

Comment:
we could also just temporarily remove this code -> add a short sleep -> assume everything succeeded while we fix this to unblock folks using this script

Reply:
yeah, it doesn't make sense to have this code anymore if it'll never work. lets just delete it and make the script always check the eds dir

Reply:
ack - i'll make it so that we do no checking if --envoy-eds-dir isn't set (unless we're fine with hardcoding the eds dir location at yelp in this repo :p)